module/os/linux/zfs/arc_os.c

```diff
@@ -72,11 +74,19 @@
  * See also the comment in arc_shrinker_count().
  * Set to 0 to disable limit.
  */
-int zfs_arc_shrinker_limit = 10000;
+static int zfs_arc_shrinker_limit = 10000;
+
+/*
+ * Relative cost of ARC eviction, AKA number of seeks needed to restore evicted
+ * page.  Bigger values make ARC more precious and evictions smaller comparing
+ * to other kernel subsystems.  Value of 4 means parity with page cache.
+ */
+static int zfs_arc_shrinker_seeks = DEFAULT_SEEKS;
```

Comment:
Where does `4` come from in the comment? I'm guessing it falls out of the math in `do_shrink_slab()`?

Reply:
Yes.  From my code reading it seems kernel applies memory pressure to page cache in a ` >> priority` fraction of size.  Same time `do_shrink_slab()` multiples it by 4 before passing it to `->scan_objects()`.